kube/services/fence/fence-deploy.yaml

```diff
@@ -19,6 +19,18 @@ spec:
         app: fence
         GEN3_DATE_LABEL
     spec:
+      affinity:
```

Comment:
this is trying to make pods for particular app don't be scheduled on the same node?

Reply:
yep